docs/api.md

```diff
@@ -318,6 +318,7 @@ back to the client. This code will be traded for a token at the
 - `assertion`: A FxA assertion for the signed-in user.
 - `state`: A value that will be returned to the client as-is upon redirection, so that clients can verify the redirect is authentic.
 - `response_type`: Optional. If supplied, must be either `code` or `token`. `code` is the default. `token` means the implicit grant is desired, and requires that the client have special permission to do so.
+- `ttl`: Optional if `response_type=token`, forbidden if `response_type=code`. Indicates the lifespan in seconds for the requested implicit grant token. If supplied, it must be a positive number less than or equal to 1,209,600 (2 weeks).
```

Comment:
I expect we will reduce this significantly in future, so maybe not worth calling it out in the docs.  Rather we can say that the server may cap this to an internal maximum value, and clients should check the `expiresIn` return value for the actual TTL.